packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -224,17 +222,23 @@ $expand-less-icon: map-merge(
       }
     }
 
-    &[aria-expanded="false"] {
+    &[aria-expanded] {
       span {
-        @include place-icon($-add-icon, "after");
-
         &::after {
           position: absolute;
+          top: 50%;
           right: 0;
+          transform: translateY(-50%);
+        }
+      }
+    }
```

Comment:
## `[aria-expanded]` vs accordion-button

Originally [commented](https://github.com/uswds/uswds/pull/5655#discussion_r1419687891) on #5655 

Im curious if it would make more sense if we moved these lines to `usa-nav__primary.usa-accordion__button.span` on line 300 instead of `[aria-expanded]`. Seems like it might be a bit more descriptive but it is further away from the other `aria-expanded=true/false` styles which are relevant to this work.

https://github.com/uswds/uswds/blob/05e4ee234628ca6f45a3ca76ef1e470a02a0add3/packages/usa-nav/src/styles/_usa-nav.scss#L307-L315

Reply:
In future work this type of research should be part of the initial stages. This will help us write more maintainable code and robust features.